addon/lib/utils/register-helper.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+function registerHelperIteration1(name, helperFunction) {
+  //earlier versions of ember with htmlbars used this
+  Ember.HTMLBars.helpers[name] = helperFunction;
```

Comment:
@stefanpenner So it looks like https://github.com/emberjs/list-view/blob/master/addon/helper.js#L8 is missing this one. Should we add it to list-view as well ?